atomate2/openmm/jobs/core.py

```diff
@@ -0,0 +1,147 @@
+
+from typing import List, Union, Optional, Dict
+from pathlib import Path
+
+from dataclasses import dataclass
+
+from jobflow import Maker
+
+from pymatgen.io.openmm.schema import InputMoleculeSpec
+from pymatgen.io.openmm.sets import OpenMMSet
+
+from atomate2.openmm.jobs.base import BaseOpenmmMaker
+
+
+@dataclass
+class InputMaker(Maker):
+    name: str = "input maker"
+
```

Comment:
I _think_ the `@job` decorator is needed  for Job makers

Reply:
Good catch, this should have the @job decorator.